### PR TITLE
Add blog for CFP announcement 2024

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**

### CODE REVIEW

File: themes/inpycon2023/static/images/logo_2024.png
Comment (anistark) [inline themes/inpycon2023/static/images/logo_2024.png:1]: I believe we need to make a separate folder for `inpycon2024`. Should touch `inpycon2023` theme.

**Review by anistark — CHANGES_REQUESTED** [review]

File: themes/inpycon2023/static/images/logo_2024.png
Comment (Sc0rpi0n101) [reply themes/inpycon2023/static/images/logo_2024.png:1]: Yes, but it takes the static resources from the current theme. Until we have a theme for 2024 and update the blog to that theme, we have to keep it in 2023 theme.

Comment (anistark) [reply themes/inpycon2023/static/images/logo_2024.png:1]: Yeah so we can copy inpycon2023 to inpycon2024 with same resources. When we've the theme ready, it's replacing inside inpycon2024. We won't need to touch inpycon2023 at all.

File: content/blog/2024/cfp-logo-announcement.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ We are excited to announce that [PyCon India 2024](https://in.pycon.org/2024/) i
...
-So, whether you are submitting a proposal for talks for the first time or just want to share what you have been up to with Python for a long time. This is YOUR time to shine!
+So, whether you are submitting a proposal for talks for the first time or just want to share what you have been doing with Python for a long time, this is YOUR time to shine!
 
 Please make a note of some important dates as mentioned below:
 
 - CFP opens on **10th March 2024**
 - CFP closes on **31st May 2024**
-The schedule shall be released on the Last week of July
+- The schedule shall be released on **1st September 2024**
 - Talks will be presented on **20th - 23rd September 2024**
```
Comment (anistark) [inline content/blog/2024/cfp-logo-announcement.md:27]: Talks are presented 21st - 22nd September only.
Workshop on 20th.

File: content/blog/2024/cfp-logo-announcement.md [hunk 2]
```diff
@@ -39,10 +39,10 @@ As you think about the experiences, skills, and knowledge you want to bring, We
 When submitting a proposal, you must fill in the following details.
 
 1. Title
-Give your talk/workshop a name that accurately describes the focus of the session for potential attendees.
+Give your talk/workshop a name that accurately describes the session's focus for potential attendees.
 
 2. Description
-A brief description of the talk/workshop is needed here. Provide as much detail as you can to guide us through your plans to cover the topic. The workshop duration needs to be 2.5 hours, and the talk duration needs to be 30 minutes. Hence, a rough timeline detailing how you will spend time explaining the concept would be a great idea.
+A brief description of the talk/workshop is needed here. Provide as much detail as possible to guide us through your plans to cover the topic. The workshop should last 2.5 hours, and the talk should last 30 minutes. Hence, a rough timeline detailing how you will spend time explaining the concept would be great.
```
Comment (anistark) [inline content/blog/2024/cfp-logo-announcement.md:45]: Isn't workshop 3 hours total?

**Review by anistark — CHANGES_REQUESTED** [review]

File: content/blog/2024/cfp-logo-announcement.md (hunk 2 above)
Comment (anistark) [reply content/blog/2024/cfp-logo-announcement.md:45]: We can perhaps add `(A small break in between as planned by the speaker)`

**Review by anistark — APPROVED** [review]

**Pull request merged**